JustSaying.IntegrationTests/JustSayingFluently/MultiRegion/WithSqsPointToPointSubscriber/WhenAFailoverRegionIsSetup.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using JustSaying.Messaging;
```

Comment:
Can we put the extension method in the same namespace as the publish method so it's source code compatible?

Reply:
I would like to make that easier. Right now this PR has a bout 20 files doing `using JustSaying.Messaging;` so that they can use these overloads/extensions. But it is in the same namespace as `IMessagePublisher.PublishAsync`